src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/ExceptionManagerEndpoint.java

```diff
@@ -30,9 +33,22 @@ public ResponseEntity<String> getBadMessagesSummary(
     userIdentity.checkGlobalUserPermission(
         userEmail, UserGroupAuthorisedActivityType.EXCEPTION_MANAGER_VIEWER);
 
+    if (Integer.parseInt(exceptionManagerClient.getBadMessagesCount()) > exceptionCountThreshold) {
+      return new ResponseEntity<>("", HttpStatus.OK);
+    }
+
```

Comment:
This doesn't feel like the right thing to do. When it's over the threshold, never appears on the page and it just looks like the exception manager is broken

Reply:
I think this should do what you want
```suggestion
    if (Integer.parseInt(exceptionManagerClient.getBadMessagesCount()) > exceptionCountThreshold) {
      return new ResponseEntity<>("[]", HttpStatus.OK);
    }

```